### ISSUE TITLE
Diskussion Aufgabe 5

### ISSUE THREAD

**Issue by Hutan92** [Sarvin23/Titanic#4]
Hey Leute, Goltan und ich haben bei Aufgabe 4 einige Analysen und Visualisierungen durchgeführt und erstellt , habt ihr vielleicht noch eine Idee beziehungsweise Verbesserungsvorschläge?

**Comment by Goli2002** [comment]
Genau bei Aufgabe 5, könnt ihr noch weitere Ideen nennen, falls euch etwas einfällt.

**Comment by Sumed22** [comment]
Ich finde es sehr interessant dass die Output bei Bivariate Stats für Zusammenhang von metrischen und dicothomen Variablen nicht ganz besonders stark. Vielleicht ist es besser, dass wir auch den Zusammenhang von kategorischen und dicothomen Variablen betrachten.

**Comment by Sumed22** [comment]
Aber basierend auf den Bivariate Statistiken von den Variablen Age und  Survived, mit dem Wert -0.69, kann man sagen, dass die Wahrscheinlichkeit ist geringer , dass ältere Menschen den Unfall überleben. Aber dieser Wert ist sehr niedrig, deswegen ist die Linearität nicht ganz genau.

**Comment by Goli2002** [comment]
> Aber basierend auf den Bivariate Statistiken von den Variablen Age und Survived, mit dem Wert -0.69, kann man sagen, dass die Wahrscheinlichkeit ist geringer , dass ältere Menschen den Unfall überleben. Aber dieser Wert ist sehr niedrig, deswegen ist die Linearität nicht ganz genau.

Kann auch daran liegen, dass der durchschnittliche Alter etwas bei 29 Jahren liegt in unserem Fall.

**Comment by Hutan92** [comment]
Es gibt sehr viele verschiedene Aspekte die man betrachten muss, aber auch verschiedene Korrelationen.
Man erkennt aufjedenfall einige Dinge , jedoch denke ich die Ergebnisse sind nicht zu 100% eindeutig.

**Comment by Sumed22** [comment]
Ja, das stimmt auch.

**Comment by Goli2002** [comment]
> Es gibt sehr viele verschiedene Aspekte die man betrachten muss, aber auch verschiedene Korrelationen. Man erkennt aufjedenfall einige Dinge , jedoch denke ich die Ergebnisse sind nicht zu 100% eindeutig.

Ja sehe ich auch so, die Überlebensrate hat mit verschiedenen anderen Faktoren einen Zusammenhang. Ein Beispiel wäre, dass es  eine sichtbare Korrelation zwischen der Klasse und der Überlebensrate gibt.  Passagiere der ersten Klasse hatten eine höhere Überlebensrate als Passagiere der zweiten und dritten Klasse.

**Comment by Goli2002** [comment]
Ja, und sie variiert auch je nach Geschlecht, wobei Frauen eine höhere Überlebensrate als Männer aufweisen.

**Comment by Hutan92** [comment]
Ja, und sie variiert auch je nach Geschlecht, wobei Frauen eine höhere Überlebensrate als Männer aufweisen.

**Comment by Goli2002** [comment]
Ich würde sagen alles in einem, haben wir den Datensatz gut vorbereitet und auch hilfreiche Ergebnisse erzielt.

**Comment by Goli2002** [comment]
Findet ihr das auch?

**Comment by Sumed22** [comment]
Ja , wir haben sehr gut gemacht. Ich bin stolz auf uns
